bigbluebutton-html5/imports/api/acl/Acl.js

```diff
@@ -12,11 +13,22 @@ export class Acl {
     check(permission, String);
     const permissions = this.getPermissions(credentials);
 
-    if (permissions) {
-      return this.fetchPermission(permission, permissions);
-    }
+    return this.checkToken(credentials) && this.fetchPermission(permission, permissions);
+  }
 
-    return false;
+  checkToken(credentials) {
+    // skip token check in client `can` calls since we dont have the authToken in the collection
+    if (!Meteor.isServer) return true;
+
+    const { meetingId, requesterUserId: userId, requesterToken: authToken } = credentials;
+
+    const User = this.Users.findOne({
+      meetingId,
+      userId,
+      authToken,
+    });
+
+    return !!User; // if he found a user means the meeting/user/token is valid
```

Comment:
I know that this `!!` works just fine, but oh boy is this obscure! I personally prefer `Boolean(User)`. Airbnb Style Guide approves both though: [#type-casting--coercion](https://github.com/airbnb/javascript/tree/es5-deprecated/es5#type-casting--coercion). Not a big deal, can be left as it is now.